### ISSUE TITLE
critic activation function may be wrong

### ISSUE THREAD

**Issue by yuh8** [OctThe16th/PPO-Keras#13]
tanh activation is used for critic output which limits the output range between -1 and 1. However, the self.reward which is used as ground truth for critic training is discounted cumulative sum of all rewards which can easily exceed 1. Is tanh a sensible activation for loss?
